python-spec/src/somacore/experiment.py

```diff
@@ -65,8 +63,14 @@ class Experiment(
     ms = _mixin.item[_MeasColl]()
     """A collection of named measurements."""
 
-    spatial = _mixin.item[_SceneColl]()
-    """TODO Add docs for spatial collecion."""
+    spatial = _mixin.item[_SceneColl]()  # TODO: Discuss the name of this element.
```

Comment:
> 1. In spatial soma, is `spatial` a collection used to access a Scene object?

Yes. More precisely it is the `Collection` that the scene objects are contained inside of. 

> 2. If so, what is the key used to access a `Scene` in container? experiment.spatial[???] - Here the key is of type string right? Maybe a concrete example would help.

The `Scene` name is arbitrary and up to the user, just like for a `Measurement`. For example, for a collection of Visium datasets, it would make sense to use the `dataset_id` for the `Scene` name.

Reply:
@prathapsridharan and @jp-dark We discussed in our last meeting that and integer may be better, and in my view we settled on using integers for `scene_id`

Reply:
After a discussion we settled on strings